bitwarden_license/src/Sso/Controllers/AccountController.cs

```diff
@@ -297,9 +298,10 @@ await AutoProvisionUserAsync(
 
         if (preventOrgUserLoginIfStatusInvalid)
         {
-            if (user == null) throw new Exception(_i18nService.T("UserShouldBeFound"));
+            User guaranteedUserWithOrgUser = possibleUserLinkedWithSso
```

Comment:
🎨 : would something like `resolvedUser` be better here? The org user relationship is true, but we can let the `orgUser` var in the same context imply the org user existence?